backend/models/user.py

```diff
@@ -0,0 +1,9 @@
+from sqlalchemy import Column, Integer, String
+from database import Base
+
+
+class User(Base):
+    __tablename__ = "users"
+    id = Column(Integer, primary_key=True)
+    email = Column(String, nullable=False, index=True)
```

Comment:
Now that we removed email as primary key, can it be duplicated across users?

Reply:
I could be if directly insert through database. But if you insert through API I have logic to check if there is existing email.
And yes it would not be good if it can be duplicated so I solved by add `UNIQUE` constraint to email column.